pkg/connectorrunner/runner.go

```diff
@@ -216,6 +216,9 @@ type rotateCredentialsConfig struct {
 	resourceType string
 }
 
+type eventStreamConfig struct {
```

Comment:
Just placeholder for when we might eventually need a config?

Reply:
Yep, and follows the same format as the other runners.